src/containers/ProfileDetails/components/CaptivePortal/index.js

```diff
@@ -269,14 +228,16 @@ const CaptivePortalForm = ({
   };
 
   const handleOnWhitelist = value => {
-    validateWhitelist(null, value).then(() => {
-      setWhitelist([...whitelist, value.toLowerCase().trim()]);
-      setWhitelistSearch('');
-      setWhitelistValidation({
-        status: null,
-        help: null,
-      });
-    });
+    validateWhitelist(null, value)
+      .then(() => {
+        setWhitelist([...whitelist, value.toLowerCase().trim()]);
+        setWhitelistSearch('');
+        setWhitelistValidation({
+          status: null,
+          help: null,
+        });
+      })
+      .catch(() => {});
```

Comment:
We have all the descriptive errors when the promise is rejected but it doesn't seem like we do anything with them. Does the error get displayed anywhere? If not, I think giving the user the error would make sense

Reply:
Yes, the errors are shown from the catch condition in the `handleOnChangeWhitelist` function.   The `handleOnWhitelist` function above is when the user actually adds the whitelist item.